.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: Run Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js and Corepack
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.x
```

Comment:
should we use lts ? 22x?

Reply:
Agree, i just picked the same version as in the others workflows. We can follow up updating the version in other PR